go.mod

```diff
@@ -130,33 +134,44 @@ require (
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	k8s.io/cloud-provider v0.28.9 // indirect
-	k8s.io/csi-translation-lib v0.28.9 // indirect
-	k8s.io/dynamic-resource-allocation v0.28.9 // indirect
-	k8s.io/kube-scheduler v0.28.9 // indirect
-	k8s.io/mount-utils v0.28.9 // indirect
+	k8s.io/cloud-provider v0.30.8 // indirect
+	k8s.io/csi-translation-lib v0.30.8 // indirect
+	k8s.io/dynamic-resource-allocation v0.30.8 // indirect
+	k8s.io/kube-scheduler v0.30.8 // indirect
+	k8s.io/mount-utils v0.30.8 // indirect
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
 replace (
 	golang.org/x/sys => golang.org/x/sys v0.19.0
-	k8s.io/api => k8s.io/api v0.28.9
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.28.9
-	k8s.io/apimachinery => k8s.io/apimachinery v0.28.9
-	k8s.io/apiserver => k8s.io/apiserver v0.28.9
-	k8s.io/client-go => k8s.io/client-go v0.28.9
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.28.9
-	k8s.io/code-generator => k8s.io/code-generator v0.28.9
-	k8s.io/component-base => k8s.io/component-base v0.28.9
-	k8s.io/component-helpers => k8s.io/component-helpers v0.28.9
-	k8s.io/controller-manager => k8s.io/controller-manager v0.28.9
-	k8s.io/cri-api => k8s.io/cri-api v0.28.9
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.28.9
-	k8s.io/dynamic-resource-allocation => k8s.io/dynamic-resource-allocation v0.28.9
-	k8s.io/kms => k8s.io/kms v0.28.9
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.28.9
-	k8s.io/kubelet => k8s.io/kubelet v0.28.9
-	k8s.io/mount-utils => k8s.io/mount-utils v0.28.9
+	k8s.io/api => k8s.io/api v0.30.8
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.30.8
+	k8s.io/apimachinery => k8s.io/apimachinery v0.30.8
+	k8s.io/apiserver => k8s.io/apiserver v0.30.8
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.30.8
```

Comment:
why add these replacements, they're not used anywhere 

1.  k8s.io/cli-runtime
2. k8s.io/endpointslice
3. k8s.io/kubelet
4. k8s.io/legacy-cloud-providers
5. pod-security-admission
6. sample-apiserver

Reply:
refer: https://github.com/kubernetes/kubernetes/issues/79384
I attempted to generate it using the -p (prune) option, but these packages are still required. If I delete them manually, errors such as "unknown revision v0.0.0" will occur.